canonical/greenhouse-browser-scripts
# Add the todo expander user plugin

## Done

Added the todo expander script and documented it.

## QA

1. Install the new script
2. Open greenhouse on a list of candidates
3. See that new button
4. Click it and see all the todo sections are revealled
